index.js

```diff
@@ -36,7 +60,9 @@ function getSourceLocation(sourceLocator) {
   if (!sourceLocator)
     return null;
 
-  const pnp = require(`pnpapi`);
+  const pnp = getPnpApi();
+  if (!pnp) 
+    return null;
```

Comment:
Doesn't that risk being a problem, if it just always return the main process' API?